torch/export/graph_signature.py

```diff
@@ -31,6 +31,7 @@ class SymIntArgument:
 @dataclasses.dataclass
 class CustomObjArgument:
     name: str
+    class_fqn: str
```

Comment:
is this fqn defined in the context of Python? If so do we have requirements for the backends to properly decode this fqn?

Reply:
No, this is not tied to python. 

In C++, we are also relying on this fqn to get handle to the CustomClassHolder. 

When user implements a CustomClassHolder class, framework will assign a fqn for this class.